worldedit-core/src/main/java/com/sk89q/worldedit/world/block/BlockCategories.java

```diff
@@ -32,8 +32,8 @@
     public static final BlockCategory BIRCH_LOGS = register("minecraft:birch_logs");
     public static final BlockCategory BUTTONS = register("minecraft:buttons");
     public static final BlockCategory CARPETS = register("minecraft:carpets");
-    public static final BlockCategory CORAL = register("minecraft:coral");
-    public static final BlockCategory CORAL_PLANTS = register("minecraft:coral_plants");
+    public static final BlockCategory CORALS = register("minecraft:corals");
```

Comment:
Was this generated using MCUtils? It's very odd that it has changed. If this has changed across versions we may need to rethink how it works :(

Reply:
done by hand since ##coral was returning 0 blocks and i checked the wiki. it's very possible that other things have/will change. (I was just using coral for testing since it's conveniently waterlogged by default.) definitely agree it would be nice to do this some other way (can we get all valid tags from the server on platform init?)